recipes/mender-monitor/debian-master/mender-monitor.install

```diff
@@ -24,5 +24,6 @@ usr/share/mender-monitor/lib/mender-client-lib.sh
 usr/share/mender-monitor/lib/monitor-lib.sh
 usr/share/mender-monitor/lib/service-lib.sh
 usr/share/mender-monitor/lib/subsystem-storage-lib.sh
+usr/share/mender-monitor/lib/time-lib.sh
```

Comment:
Can we consider adding the whole folder, or is that bad practice? It is namespaced under mender-monitor.

Reply:
I don't think that it is a bad practice, and if so we are already doing it in other recipes. I'll experiment with this separately, together with @merlin-northern improvements suggestions